### DISCUSSION TITLE
[4.0] Dragging new modules around

### DISCUSSION THREAD

**Discussion by tonypartridge** [joomla/joomla-cms#47172] [2. Cold Features]
Given we can add new modules to the dashboard it would make sense to be able to drag them around as we want them?

**Comment by brianteeman** [comment]
Go for it. we have the dragula library in core to help you do it

**Comment by infograf768** [comment]
Modified title to be informative...

**Comment by chmst** [comment]
Would be highly appreciated!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28606">issues.joomla.org/tracker/joomla-cms/28606</a>.</sub>

**Comment by micker** [comment]
Cool feature... Need to add full width too !
https://github.com/joomla/joomla-cms/issues/25952

**Comment by micker** [comment]
@C-Lodder have added it in his template
